armi/utils/tests/test_plotting.py

```diff
@@ -129,8 +120,6 @@ def test_plotCartesianBlock(self):
 
     def _checkExists(self, fName):
         self.assertTrue(os.path.exists(fName))
-        if self.removeFiles:
-            os.remove(fName)
 
```

Comment:
Since you are removing `self.removeFiles`, it looks like we can remove its declaration at the top of the Class.

Reply:
Good catch!  Fixed.